dataland-frontend/src/components/general/TheNavi.vue

```diff
@@ -0,0 +1,83 @@
+<template>
```

Comment:
Currently commented out. Move to another feature branch?

Reply:
Yes, we could create a different feature branch with all the components not included in this one.

Reply:
Sounds good.  Keeping this comment open until it has been moved to a different branch,